addons/mrp/models/stock_warehouse.py

```diff
@@ -276,3 +276,15 @@ def _update_name_and_code(self, name=False, code=False):
             if warehouse.manufacture_pull_id and name:
                 warehouse.manufacture_pull_id.write({'name': warehouse.manufacture_pull_id.name.replace(warehouse.name, name, 1)})
         return res
+
+class Orderpoint(models.Model):
+    _inherit = "stock.warehouse.orderpoint"
+
+    @api.constrains('product_id')
+    def check_product_is_not_kit(self):
+        for orderpoint in self:
+            product = orderpoint.product_id
+            if self.env['mrp.bom'].search(['|', ('product_id', '=', product.id),
```

Comment:
You can use `_bom_find` with the param `bom_type='phantom'`.

Reply:
@ryv-odoo Either I'm missing something, or the param `bom_type` is not available in 12.0
(I will change this search on 13.0)

Reply:
Oups, you are perfectly right, it doesn't exist in 12.0. My bad.